Source/ACE/Entity/WorldObject.cs

```diff
@@ -1456,13 +1456,19 @@ internal void SetInventoryForWorld(WorldObject inventoryItem)
 
             inventoryItem.ContainerId = null;
             inventoryItem.WielderId = null;
+            // TODO: create enum for this once we understand this better.
+            // This is needed to make items lay flat on the ground.
+            inventoryItem.AnimationFrame = 0x65;
+            inventoryItem.WeenieFlags = inventoryItem.SetWeenieHeaderFlag();
```

Comment:
It is automatic on the write out, but not sure if it updates the object in the interim.   I would like to leave this in for a bit, I will put a TODO to review for removal if that is OK?